engine/gui/src/gui.cpp

```diff
@@ -4443,6 +4443,12 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
             SetNodeSpineScene(scene, *out_node, GetNodeSpineScene(scene, node), GetNodeSpineSkin(scene, node), GetNodeSpineAnimation(scene, node), false);
         }
 
+        if (n->m_Node.m_FlipbookAnimHash != 0)
+        {
+            float playback_rate = GetNodeFlipbookPlaybackRate(scene, node);
+            PlayNodeFlipbookAnim(scene, *out_node, n->m_Node.m_FlipbookAnimHash, 0.0f, playback_rate, 0, 0, 0);
```

Comment:
I would assume that a clone should copy the current time + playback mode as well?

Reply:
Perhaps current time, but playback mode can't be changed at runtime?